src/zenml/integrations/great_expectations/materializers/ge_materializer.py

```diff
@@ -122,13 +123,90 @@ def save(self, obj: SerializableDictDot) -> None:
             obj: A Great Expectations object.
         """
         filepath = os.path.join(self.uri, ARTIFACT_FILENAME)
-        artifact_dict = obj.to_json_dict()
+        artifact_dict = self.serialize_ge_object(obj)
```

Comment:
Why are we jumping through these hoops to serialize/deserialize these objects? Doesn't GreatExpectations have a built-in way to handle these types.

Reply:
my bad in the description. its still checkpointresult but the class has been updated apparently so the earlier to_dict function doesn't work anymore